content/07-native-tokens/03-getting-started.mdx

```diff
@@ -79,19 +75,19 @@ The `cardano-cli` can specify multi-asset values in transaction outputs and when
 
 - ada values are defined as integer (INT) lovelace, e.g. `42 lovelace`
 - multi-asset values can be defined as:
-  - `INT policyid.assetname`, e.g. `42 $MYPOLICY.myassetname`
+  - `INT policyid.assetName(hexidecimal format)`, e.g. `42 $MYPOLICY.assetName(hexidecimal format)`
```

Comment:
Would be good to use an example of an actual hex**a**decimal value anywhere we do `e.g.`

Reply:
Whatever the hex value is for the chosen token name